### PR TITLE
feat(hints): add NewHint#26

### PR DESCRIPTION
**Pull request by MegaRedHand**
## Description

Depends: #990

This PR adds the missing hint _NewHint#26_, now known as `VERIFY_ZERO_EXTERNAL_SECP`.

### CODE REVIEW

**Review by Oppen — APPROVED** [review]

File: src/hint_processor/builtin_hint_processor/secp/field_utils.rs [hunk 1]
```diff
@@ -154,38 +154,27 @@ mod tests {
...
+        ];
+        for hint_code in hint_codes {
+            let mut vm = vm_with_range_check!();
+            //Initialize run_context
+            run_context!(vm, 0, 9, 9);
+            //Create hint data
+            let ids_data = non_continuous_ids_data![("val", -5), ("q", 0)];
+            vm.segments = segments![((1, 4), 0), ((1, 5), 0), ((1, 6), 0)];
+            //Execute the hint
+            assert_matches!(
```
Comment (Oppen) [inline src/hint_processor/builtin_hint_processor/secp/field_utils.rs:170]: You can fix this alert changing the `assert_matches!(foo, Ok(()))` to `assert!(foo.is_ok())`.

**Review by Oppen — APPROVED** [review]

**Review by Oppen — APPROVED** [review]

File: src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs [hunk 2]
```diff
@@ -249,7 +249,7 @@ impl HintProcessor for BuiltinHintProcessor {
             hint_code::BLAKE2S_COMPUTE => {
                 compute_blake2s(vm, &hint_data.ids_data, &hint_data.ap_tracking)
             }
-            hint_code::VERIFY_ZERO_V1 | hint_code::VERIFY_ZERO_V2 => {
+            hint_code::VERIFY_ZERO_V1 | hint_code::VERIFY_ZERO_V2 | hint_code::VERIFY_ZERO_V3 => {
```
Comment (pefontana) [inline src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs:252]: Actually, here we shouldnt execute the same functions.
It is just a minor detail but see that this hint you are implementing doesn't import the SECP constant, so in this cases, we should get the SECP constant from the scope

Comment (MegaRedHand) [reply src/hint_processor/builtin_hint_processor/builtin_hint_processor_definition.rs]: Done!

**Comment by MegaRedHand** [comment]
Rebased to main and addressed comment in https://github.com/lambdaclass/cairo-rs/pull/992/commits/67477d98ed9f1a55104c7ca8154988092ed12e59

**Review by Oppen — APPROVED** [review]

**Review by pefontana — APPROVED** [review]

**Pull request merged**